stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/MessageListItemViewHolderFactory.kt

```diff
@@ -15,87 +25,89 @@ import io.getstream.chat.android.ui.messages.adapter.viewholder.PlainTextWithMed
 import io.getstream.chat.android.ui.messages.adapter.viewholder.ThreadSeparatorViewHolder
 import io.getstream.chat.android.ui.messages.adapter.viewholder.decorator.DecoratorProvider
 
-public open class MessageListItemViewHolderFactory(
-    private val decoratorProvider: DecoratorProvider
-) {
+public open class MessageListItemViewHolderFactory {
+
+    internal lateinit var decoratorProvider: DecoratorProvider
```

Comment:
Injected the same way as listenerContainer now, so that it isn't a required constructor parameter (which clients couldn't provide if they extended this class)